mitodl/open-discussions
# fix create post bug

#### What are the relevant tickets?

none

#### What's this PR do?

Fixes a bug with the create post page, from the introduction of the little 'X' on the input.

#### How should this be manually tested?

Repro the bug on master first:

- go to `/create_post`
- click on 'Write Something' or 'Share a Link'
- you should see an exception in the console
- it should let you continue if you pick a subreddit first

Then check this out and confirm it fixes the issue.
